### PR TITLE
Updating CAPI Overview - Prereqs Section/Additional Links

### PR DESCRIPTION
**Pull request by sunilarjun**
Adding an update to the CAPI Overview page "Prerequisites" section specifying where to disable and remove webhooks and the embedded-cluster-api, as well as adding links to the CAPI site on CAPI provider information.

### CODE REVIEW

File: docs/integrations-in-rancher/cluster-api/overview.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ Below is a visual representation of the key components of Rancher Turtles and th
 
 ![overview](/img/30000ft_view.png)
 
+## Security
+
+[SLSA](https://slsa.dev/spec/v1.0/about) is a set of incrementally adoptable guidelines for supply chain security, established by industry consensus. The specification set by SLSA is useful for both software producers and consumers: producers can follow SLSA’s guidelines to make their software supply chain more secure, and consumers can use SLSA to make decisions about whether to trust a software package.
+
+Rancher Turtles meets [SLSA Level 3](https://slsa.dev/spec/v1.0/levels#build-l3) requirements for appropriate build platform, consistent build process, and provenance distribution. For more information, visit the [Rancher Turtles Security](https://turtles.docs.rancher.com/docs/security/slsa) document.
```
Comment (btat) [inline docs/integrations-in-rancher/cluster-api/overview.md:19]:
```suggestion
Rancher Turtles meets [SLSA Level 3](https://slsa.dev/spec/v1.0/levels#build-l3) requirements for appropriate build platform, consistent build process, and provenance distribution. For more information, visit the [Rancher Turtles Security](https://turtles.docs.rancher.com/security/slsa) document.
```

File: versioned_docs/version-2.7/integrations-in-rancher/cluster-api/overview.md [hunk 2]
```diff
@@ -12,6 +12,12 @@ Below is a visual representation of the key components of Rancher Turtles and th
 
 ![overview](/img/30000ft_view.png)
 
+## Security
+
+[SLSA](https://slsa.dev/spec/v1.0/about) is a set of incrementally adoptable guidelines for supply chain security, established by industry consensus. The specification set by SLSA is useful for both software producers and consumers: producers can follow SLSA’s guidelines to make their software supply chain more secure, and consumers can use SLSA to make decisions about whether to trust a software package.
+
+Rancher Turtles meets [SLSA Level 3](https://slsa.dev/spec/v1.0/levels#build-l3) requirements for appropriate build platform, consistent build process, and provenance distribution. For more information, visit the [Rancher Turtles Security](https://turtles.docs.rancher.com/docs/security/slsa) document.
```
Comment (btat) [inline versioned_docs/version-2.7/integrations-in-rancher/cluster-api/overview.md:19]:
```suggestion
Rancher Turtles meets [SLSA Level 3](https://slsa.dev/spec/v1.0/levels#build-l3) requirements for appropriate build platform, consistent build process, and provenance distribution. For more information, visit the [Rancher Turtles Security](https://turtles.docs.rancher.com/security/slsa) document.
```

File: versioned_docs/version-2.8/integrations-in-rancher/cluster-api/overview.md [hunk 3]
```diff
@@ -12,6 +12,12 @@ Below is a visual representation of the key components of Rancher Turtles and th
 
 ![overview](/img/30000ft_view.png)
 
+## Security
+
+[SLSA](https://slsa.dev/spec/v1.0/about) is a set of incrementally adoptable guidelines for supply chain security, established by industry consensus. The specification set by SLSA is useful for both software producers and consumers: producers can follow SLSA’s guidelines to make their software supply chain more secure, and consumers can use SLSA to make decisions about whether to trust a software package.
+
+Rancher Turtles meets [SLSA Level 3](https://slsa.dev/spec/v1.0/levels#build-l3) requirements for appropriate build platform, consistent build process, and provenance distribution. For more information, visit the [Rancher Turtles Security](https://turtles.docs.rancher.com/docs/security/slsa) document.
```
Comment (btat) [inline versioned_docs/version-2.8/integrations-in-rancher/cluster-api/overview.md:19]:
```suggestion
Rancher Turtles meets [SLSA Level 3](https://slsa.dev/spec/v1.0/levels#build-l3) requirements for appropriate build platform, consistent build process, and provenance distribution. For more information, visit the [Rancher Turtles Security](https://turtles.docs.rancher.com/security/slsa) document.
```

**Review by btat — CHANGES_REQUESTED** [review]
There are a couple other links to the Turtles site that aren't part of this PR's diff that needs to have the `/docs` removed.

File: docs/integrations-in-rancher/cluster-api/overview.md [hunk 4]
```diff
@@ -21,6 +27,35 @@ To simplify setting up Rancher for installing Rancher Turtles, the official Ranc
 - Disables the `embedded-cluster-api` feature in Rancher.
 - Deletes the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks, as they are no longer needed.
 
+The above webhooks can be removed through the Rancher UI as well and are accessed from the left-hand navigation from your local cluster under the **More Resources** > **Admission** section or found by using the **Resource Search** field and inputting the webhook names. Additionally, the following `kubectl` commands can also be used to remove the necessary webhooks:
```
Comment (martyav) [inline docs/integrations-in-rancher/cluster-api/overview.md:30]:
```suggestion
These webhooks can be removed through the Rancher UI as well:

1. In the upper left corner, click **☰** > **Cluster Management**.
1. Select your local cluster.
1. In the left-hand navigation menu, select **More Resources** > **Admission**.

They can also be accessed by entering the names of the webhooks into the **Resource Search** field.

The following `kubectl` commands can manually remove the necessary webhooks:
```

The instructions need one more step describing what to click to remove the webhooks after the user accesses Admission.

**Review by martyav — CHANGES_REQUESTED** [review]

File: docs/integrations-in-rancher/cluster-api/overview.md (hunk 1 above)
Comment (sunilarjun) [reply docs/integrations-in-rancher/cluster-api/overview.md]: Updated the links, thanks Billy!

File: docs/integrations-in-rancher/cluster-api/overview.md [hunk 5]
```diff
@@ -12,6 +12,12 @@ Below is a visual representation of the key components of Rancher Turtles and th
 
 ![overview](/img/30000ft_view.png)
 
+## Security
+
+[SLSA](https://slsa.dev/spec/v1.0/about) is a set of incrementally adoptable guidelines for supply chain security, established by industry consensus. The specification set by SLSA is useful for both software producers and consumers: producers can follow SLSA’s guidelines to make their software supply chain more secure, and consumers can use SLSA to make decisions about whether to trust a software package.
```
[the remark this answers is not in the document]
Comment (sunilarjun) [reply docs/integrations-in-rancher/cluster-api/overview.md]: Went with your suggestions and updated the Security section, thanks Marty!

File: docs/integrations-in-rancher/cluster-api/overview.md (hunk 4 above)
Comment (sunilarjun) [reply docs/integrations-in-rancher/cluster-api/overview.md]: Added a couple more steps to clarify where the "Delete" option can be selected along with your suggestion. Let me know if it makes sense, thanks!

File: docs/integrations-in-rancher/cluster-api/overview.md [hunk 6]
```diff
@@ -21,6 +27,45 @@ To simplify setting up Rancher for installing Rancher Turtles, the official Ranc
 - Disables the `embedded-cluster-api` feature in Rancher.
 - Deletes the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks, as they are no longer needed.
 
+These webhooks can be removed through the Rancher UI as well:
+
+1. In the upper left corner, click **☰** > **Cluster Management**.
+1. Select your local cluster.
+1. In the left-hand navigation menu, select **More Resources** > **Admission**.
+1. From the dropdown you will see the Resource pages for `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration`, click through both pages to remove the necessary webhooks.
```
Comment (martyav) [inline docs/integrations-in-rancher/cluster-api/overview.md:35]:
```suggestion
1. From the dropdown, select the Resource pages for `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration`.
```

File: docs/integrations-in-rancher/cluster-api/overview.md [hunk 7]
```diff
@@ -21,6 +27,45 @@ To simplify setting up Rancher for installing Rancher Turtles, the official Ranc
...
+1. On the respective Resource pages, click the **⋮** that are attached to the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks to select the **Delete** option. Once deleted, you should see the webhooks removed immediately.
```
Comment (martyav) [inline docs/integrations-in-rancher/cluster-api/overview.md:36]:
```suggestion
1. On the respective Resource pages, click the **⋮** that are attached to the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks and select the **Delete** option.

 After they are deleted, you should see the webhooks removed immediately.
```

File: docs/integrations-in-rancher/cluster-api/overview.md [hunk 8]
```diff
@@ -21,6 +27,45 @@ To simplify setting up Rancher for installing Rancher Turtles, the official Ranc
...
+
+They can also be accessed by entering the names of the webhooks into the **Resource Search** field.
```
Comment (martyav) [inline docs/integrations-in-rancher/cluster-api/overview.md:38]:
```suggestion
The webhooks can also be accessed by entering the names of the webhooks into the **Resource Search** field.
```

**Review by martyav — CHANGES_REQUESTED** [review]
Some requests for the updated instructions. Thank you for providing the extra steps.

**Comment by sunilarjun** [comment]
@martyav Just updated the steps across versions!

**Review by btat — CHANGES_REQUESTED** [review]
There are a few more broken links (`/docs` in the turtles URL) on lines 95, 180, 188.

**Comment by sunilarjun** [comment]
> There are a few more broken links (`/docs` in the turtles URL) on lines 95, 180, 188.

Apologies Billy, should all be updated now!

**Review by btat — APPROVED** [review]

**Pull request merged**